### PR TITLE
Added recipe for positional

### PR DESCRIPTION
**Pull request by pmlandwehr**
Pinging @morganfainberg in case he would like to be added as a maintainer to the `positional` builder on conda-forge, a library of community-build conda packages. The maintenance burden here is light: all testing and releases are built and deployed automatically with CI. Changes to how the package is being built can be controlled by changing the recipe which will be located at https://github.com/conda-forge/positional-feedstock shortly after this PR is finished and merged. If you aren't interested in helping maintain the build, that is entirely fine too.

### CODE REVIEW

File: recipes/positional/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pbr >=1.8
+
+  run:
+    - python
+    - pbr >=1.6
```
Comment (ocefpaf) [inline recipes/positional/meta.yaml]: Why is it `1.8` above and `1.6` here?

**Comment by pmlandwehr** [comment]
Crazily, the recipe has different requirements for the different pieces: [setup.py](https://github.com/morganfainberg/positional/blob/master/setup.py) requires pbr >= 1.8, while [requirements.txt](https://github.com/morganfainberg/positional/blob/master/requirements.txt) specifies pbr >= 1.6

**Comment by morganfainberg** [comment]
I've initiated a PR for positional to fix the setup.py and requirements.txt mis-match. 
i'll probably publish a release of positional with this fix in the next few days.

Thanks for the ping here. I'm happy to help maintain this, but do not feel as though it is super important as long as you/anyone involved here is happy to contribute fixes up to positional if needed. [Hopefully positional wont have any real change needed barring a crazy bug, since it is a trivial library]

**Comment by pmlandwehr** [comment]
@morganfainberg  ok, well - I've thrown you in as a maintainer, but it should remain pretty darn low-key. If positional goes through some major changes it'll be something to worry about. If not, it won't.

**Pull request merged**